package.json

```diff
@@ -20,15 +20,18 @@
   },
   "homepage": "https://github.com/Keeeeeeeks/something-awesome#readme",
   "dependencies": {
+    "bcryptjs": "^2.4.3",
```

Comment:
Can we use `bcrypt` instead of `bcryptjs`? Not familiar with the latter, but the former has been around longer and is more frequently updated/maintained.